[2024-07-03T23:25:44.938+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-03T23:25:45.109+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Proyecto_Final_USD_ETL.tercera_tarea manual__2024-07-03T23:25:39.797530+00:00 [queued]>
[2024-07-03T23:25:45.115+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Proyecto_Final_USD_ETL.tercera_tarea manual__2024-07-03T23:25:39.797530+00:00 [queued]>
[2024-07-03T23:25:45.116+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-07-03T23:25:45.127+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): tercera_tarea> on 2024-07-03 23:25:39.797530+00:00
[2024-07-03T23:25:45.132+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=596) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-07-03T23:25:45.133+0000] {standard_task_runner.py:63} INFO - Started process 598 to run task
[2024-07-03T23:25:45.132+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'Proyecto_Final_USD_ETL', 'tercera_tarea', 'manual__2024-07-03T23:25:39.797530+00:00', '--job-id', '96', '--raw', '--subdir', 'DAGS_FOLDER/ETL_USD.py', '--cfg-path', '/tmp/tmpfd4hyab3']
[2024-07-03T23:25:45.134+0000] {standard_task_runner.py:91} INFO - Job 96: Subtask tercera_tarea
[2024-07-03T23:25:45.164+0000] {task_command.py:426} INFO - Running <TaskInstance: Proyecto_Final_USD_ETL.tercera_tarea manual__2024-07-03T23:25:39.797530+00:00 [running]> on host 5a0cf2cc04d9
[2024-07-03T23:25:45.236+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='Proyecto_Final_USD_ETL' AIRFLOW_CTX_TASK_ID='tercera_tarea' AIRFLOW_CTX_EXECUTION_DATE='2024-07-03T23:25:39.797530+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-07-03T23:25:39.797530+00:00'
[2024-07-03T23:25:45.237+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-03T23:25:45.246+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-07-03T23:25:45.246+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo Proceso completado...']
[2024-07-03T23:25:45.255+0000] {subprocess.py:86} INFO - Output:
[2024-07-03T23:25:45.256+0000] {subprocess.py:93} INFO - Proceso completado...
[2024-07-03T23:25:45.257+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-07-03T23:25:45.257+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-03T23:25:45.274+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=Proyecto_Final_USD_ETL, task_id=tercera_tarea, run_id=manual__2024-07-03T23:25:39.797530+00:00, execution_date=20240703T232539, start_date=20240703T232545, end_date=20240703T232545
[2024-07-03T23:25:45.306+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-07-03T23:25:45.318+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-07-03T23:25:45.321+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
